packetbeat/protos/tcp/tcp.go

```diff
@@ -432,6 +436,24 @@ func newInputMetrics(id, device string) *inputMetrics {
 	return out
 }
 
+// portList returns a dash-separated list of port numbers sorted ascending. A leading
+// dash is prepended to the list if it is not empty.
+func portList(m map[uint16]protos.Protocol) string {
+	if len(m) == 0 {
```

Comment:
This should never happen in normal operation, but `buildPortsMap` does not check for zero-length port mappings. I assume that the config checks for this, but I could not find anywhere that this happens. If it doesn't, it probably should.